clients/go/coreutils/fetch_literal.go

```diff
@@ -0,0 +1,60 @@
+package coreutils
```

Comment:
For this package, I think we should start adding a top-level comments (I know we dont have it, but can you please help us set the precedent)
so essentially before the package
// extract_literal.go
// Utility methods to extract a native golang value from a given Literal.
// Usage:
// .....

Reply:
Done.